src/debugger.lisp

```diff
@@ -10,11 +10,38 @@ and printing output to the provided stream. The condition object the debugger
 was entered with and optional command arguments are available for use within
 the command itself."))
 
+(defvar *skip-debugger-prefix* nil
+  "Whether or not to avoid printing the debugger line prefix when
+  executing a command.")
```

Comment:
Please don't pad documentation strings with spaces, and try to break close to 80 chars per line - that's consistent with this repository's style.

Reply:
Oops sorry I think that was my emacs and I didn't notice it